lib/council-server/src/server/graph.rs

```diff
@@ -76,7 +77,10 @@ impl ChangeSetGraph {
         change_set_id: Id,
         node_id: Id,
     ) -> Result<HashSet<String>, Error> {
-        let change_set_graph_data = self.dependency_data.get_mut(&change_set_id).unwrap();
+        let change_set_graph_data = self
+            .dependency_data
+            .get_mut(&change_set_id)
+            .ok_or(Error::DependencyDataMissing)?;
```

Comment:
Panic seen by @sprutton1 @johnrwatson @stack72 referenced above.

Reply:
I will add some logs so we know more when the failure happens.